tests/test_incremental.py

```
@@ -0,0 +1,203 @@
+            "END IONS\n"
+        )
+        mgf_path = str(tmp_path / 'reps.mgf')
+        Path(mgf_path).write_text(mgf_content)
+
+        # Create a MaRaCluster TSV where:
+        # - rep spectrum 0 → new cluster NEW_A
+        # - rep spectrum 1 → new cluster NEW_B
+        # - new spectrum in new_data.mgf index 0 → NEW_A (merges with OLD_001)
+        # - new spectrum in new_data.mgf index 1 → NEW_C (entirely new cluster)
+        tsv_content = (
+            f"reps.mgf\t0\tNEW_A\n"
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)